### PR TITLE
Feature/component styles

### PR DESCRIPTION
**Pull request by TheoStanica**
[Issue](https://github.com/TheoStanica/mark-manager/issues/69)

### CODE REVIEW

File: client/src/redux/actions/twitterActions.js [hunk 1]
```diff
@@ -127,16 +133,144 @@ export const tweetNewMessage = ({ message }) => async (dispatch) => {
...
+    type: TWITTER_UPDATE_STREAMS,
+    payload: {
+      streams: streams,
+    },
+  });
+};
+
+export const updateUserStreamsBackend = ({ streams }) => async (dispatch) => {
+  try {
+    const updatedstreams = [...streams].map((stream) => ({
```
Comment (AdiCutitoiu) [inline client/src/redux/actions/twitterActions.js:151]: It's safe to just do `streams.map()` here, without spreading the array. `map()` leaves the array unmodified and creates a new one.

File: client/src/redux/actions/twitterActions.js [hunk 2]
```diff
@@ -1,11 +1,43 @@
...
+  TWITTER_SET_STREAM_LOADING_STATUS,
+  TWITTER_SET_STREAM_TWEETS,
+  TWITTER_UPDATE_STREAMS,
   USER_SET_MESSAGES,
 } from '../types';
+import { store } from '../store';
+
+const handleError = ({ error }) => async (dispatch) => {
+  if (
+    error &&
```
Comment (AdiCutitoiu) [inline client/src/redux/actions/twitterActions.js:17]: This is a great use case for the optional chaining operator https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

File: client/src/redux/reducers/twitterReducer.js [hunk 3]
```diff
@@ -10,7 +14,6 @@ const initialState = {
   screenName: '',
   profileImage: '',
   isConnected: false,
-  home_timeline_tweets: [],
   streams: [],
```
Comment (AdiCutitoiu) [inline client/src/redux/reducers/twitterReducer.js:17]: There's a problem with this structure: Whenever you need to modify a property from a stream, you need to create a new array of streams, which triggers a lot of unnecessary rerenders. Remind me to discuss this on Monday.
[NO ACTION REQUIRED]

File: client/src/components/Tweet/TweetMessage.js [hunk 4]
```diff
@@ -1,26 +1,26 @@
 import React from 'react';
 import twitter from 'twitter-text';
+import { StyledMessage } from './styles';
 
 const TweetMessage = ({ tweet }) => {
-  let { text, entities } = tweet;
+  let { full_text, entities } = tweet;
```
[the remark this answers is not in the document]
Comment (TheoStanica) [reply client/src/components/Tweet/TweetMessage.js]: renamed to `text` as it was before

File: client/src/redux/actions/twitterActions.js (hunk 1 above)
Comment (TheoStanica) [reply client/src/redux/actions/twitterActions.js]: Fixed. Also fixed this for loadUserStreams Action as it was the same issue

File: client/src/redux/actions/twitterActions.js (hunk 2 above)
Comment (TheoStanica) [reply client/src/redux/actions/twitterActions.js]: Fixed

**Review by RaduAlbastroiu — APPROVED** [review]

**Pull request merged**